code/game/objects/items/weapons/weaponry.dm

```diff
@@ -54,6 +54,116 @@
 	return(BRUTELOSS)
 
+var/list/highlander_claymores = list()
+/obj/item/weapon/claymore/highlander //ALL COMMENTS MADE REGARDING THIS SWORD MUST BE MADE IN ALL CAPS
+	desc = "<b><i>THERE CAN BE ONLY ONE, AND IT WILL BE YOU!!!</i></b>"
+	flags = CONDUCT | NODROP
+	block_chance = 0 //RNG WON'T HELP YOU NOW, PANSY
+	attack_verb = list("brutalized", "eviscerated", "disemboweled", "hacked", "carved", "cleaved", "gored") //ONLY THE MOST VISCERAL ATTACK VERBS
+	var/notches = 0 //HOW MANY PEOPLE HAVE BEEN SLAIN WITH THIS BLADE
+	var/announced = FALSE //If we've announced our winner
+
+/obj/item/weapon/claymore/highlander/New()
+	..()
+	START_PROCESSING(SSobj, src)
+	highlander_claymores += src
+
+/obj/item/weapon/claymore/highlander/Destroy()
+	STOP_PROCESSING(SSobj, src)
+	for(var/mob/living/simple_animal/shade/S in src)
+		S << "<span class='userdanger'>Your home breaks apart, and you with it!</span>"
+		qdel(S)
+	highlander_claymores -= src
+	return ..()
+
+/obj/item/weapon/claymore/highlander/process()
+	if(isliving(loc))
+		var/mob/living/L = loc
+		L.SetStunned(0)
+		L.SetWeakened(0)
+		L.SetParalysis(0)
+		if(L.stat == DEAD)
+			L << "<span class='userdanger'>You are unworthy.</span>"
+			L.visible_message("<span class='warning'>[L] explodes in a shower of gore!</span>")
+			var/obj/item/organ/brain/B = L.getorgan(/obj/item/organ/brain)
+			L.gib()
+			if(B)
+				qdel(B) //NO SECOND CHANCES
+		else
+			if(announced)
+				return
+			for(var/obj/item/weapon/claymore/highlander/H in highlander_claymores)
+				if(H != src)
+					return
+			announced = TRUE
+			world << "<span class='userdanger'>[L.real_name] IS THE ONLY ONE LEFT STANDING!</span>"
+			world << sound('sound/misc/highlander_only_one.ogg')
+			L << "<span class='notice'>YOU ARE THE ONLY ONE LEFT STANDING! You feel your bonds vanish - you can freely drop or sheathe your [name]! (The kilt stays, though.)</span>"
+			flags = CONDUCT
+
+/obj/item/weapon/claymore/highlander/examine(mob/user)
+	..()
+	user << "It has [notches] notches scratched into the blade."
+
+/obj/item/weapon/claymore/highlander/attack(mob/living/target, mob/living/user)
+	var/old_target_stat = target.stat
+	. = ..()
+	if(target && target.stat == DEAD && old_target_stat != DEAD && target.mind && target.mind.special_role == "highlander")
+		user.adjustBruteLoss(-200) //STEAL THE LIFE OF OUR FALLEN FOES
+		var/mob/living/simple_animal/shade/S = new(src)
+		S.real_name = "Soul of [target.real_name]"
+		S.name = S.real_name
+		target.mind.transfer_to(S)
+		S.status_flags |= GODMODE
+		S << "<span class='userdanger'>[user]'s blade has claimed your soul! Join them in glorious bloodletting!</span>"
+		add_notch(user)
+
+/obj/item/weapon/claymore/highlander/IsReflect()
+	return 1 //YOU THINK YOUR PUNY LASERS CAN STOP ME?
+
+/obj/item/weapon/claymore/highlander/proc/add_notch(mob/living/user)
+	notches++
+	force++

```

Comment:
I don't want to give a _huge_ advantage, so that everyone has a chance to get a kill or two before they get cucked.